diff -uraN fractal-10/data/resources/meson.build omv-fractal-10/data/resources/meson.build
--- fractal-10/data/resources/meson.build	2025-01-30 10:20:48.000000000 +0100
+++ omv-fractal-10/data/resources/meson.build	2025-02-09 17:14:13.131876756 +0100
@@ -9,6 +9,8 @@
   if sass_bin.found()
     # these options are not supported by grass.
     sass_options += ['--no-error-css', '--no-source-map']
+  else
+    sass_bin = find_program('sassc', required: true)
   endif
 endif
 
